daml-lf/encoder/BUILD.bazel

```diff
@@ -25,6 +25,7 @@ da_scala_library(
         "@maven//:org_scalaz_scalaz_core",
     ],
     scalacopts = lf_scalacopts,
+    tags = ["maven_coordinates=com.daml:daml-lf-encoder:__VERSION__"],
```

Comment:
@remyhaemmerle-da are you fine with publishing these libs?

Reply:
I don't want this to be published. 
I do not want anybody outside of DA to use it. 

I see two solution. 
- You generate your `dar` file in the main repo, for instance in the //daml-lf/kv-support 
- You rewrite you test in daml.

Reply:
I am very sorry for the inconvenient.